agc/asm.c

```diff
@@ -0,0 +1,399 @@
+/*
```

Comment:
Alright, I'll keep it that way for now. For most architectures this isn't an issue because you can just delegate the disassembling itself to capstone or some other lib, but I figured this is better that way.